pkg/grpc/universalapi/api_workflow.go

```diff
@@ -244,6 +248,41 @@ func (a *UniversalAPI) PurgeWorkflowAlpha1(ctx context.Context, in *runtimev1pb.
 	return emptyResponse, nil
 }
 
+// GetWorkflowBeta1 is the API handler for getting workflow details
+func (a *UniversalAPI) GetWorkflowBeta1(ctx context.Context, in *runtimev1pb.GetWorkflowRequest) (*runtimev1pb.GetWorkflowResponse, error) {
+	return a.GetWorkflowAlpha1(ctx, in)
```

Comment:
Aliasing done.
For logging, do you mean to say a single log should be printed for all workflow apis? As in if someone is calling `StartWorkflowAlpha` and `GetWorkflowAlpha` , a single log should be printed stating to migrate to beta? It can be done easily. But if you mean to say a log statement per api, it would require to declare so many `atomic.bool` variables(one for each api).

Reply:
IMO this shouldn't be shown for all workflow APIs, as their consumers are the SDKs. While its relatively low risk and controversial to add the logging message, it wouldn't be consistent with other APIs as I checked now and saw that we don't do the same thing for the configuration API. Thoughts?